docs/design/block-design.md

```diff
@@ -19,6 +19,39 @@ Basic block settings won’t always make sense in the context of the placeholder
 
 The sidebar is not visible by default on a small / mobile screen, and may also be collapsed in a desktop view. Therefore, it should not be relied on for anything that is necessary for the basic operation of the block. Pick good defaults, make important actions available in the block toolbar, and think of the sidebar as something that only power users may discover. In addition, use sections and headers in the block sidebar if there are more than a handful of options, in order to allow users to easily scan and understand the options available.
 
+## Setup state vs. live preview state
+
+Often a block will use the placeholder content to walk users through a setup process. The setup process gathers information from the user that is needed to render the block. A block’s setup state is indicated with a grey background to provide clear differentiation for the user. Not all blocks have setup states — for example, the paragraph block.
+
+
+![An example of a gallery block’s setup state on a grey background](https://make.wordpress.org/design/files/2018/12/gallery-setup.png)
```

Comment:
Only needs one newline around the image :)

Reply:
(Applies to all images in the document, fwiw, but I won't note them all as that would be tedious and annoying!)